.github/workflows/pr-validation.yaml

```diff
@@ -133,6 +133,10 @@ jobs:
         with:
           go-version-file: "go.mod"
 
+      - name: Run helm-template
+        run: |
+          helm template deploy/helm/grafana-operator/
+
```

Comment:
```suggestion
      - name: Run helm-template
        run: |
          helm template deploy/helm/grafana-operator/

      - name: Run helm-lint
        run: |
          helm lint deploy/helm/grafana-operator/

```
Let's also add `helm lint`.

Reply:
Never really seen the use of it. But sure ;)

Reply:
I think I've seen cases where template would work, but lint would fail (or vice versa), can't remember the details now. Anyway, it certainly won't hurt to have both :)